app/views/posts/show.html.erb

```diff
@@ -1,10 +1,23 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  </head>
+<div class='main-div'>
+    <div class='div'>
+            <p>Post #<%= @user.id%> By <%= @user.author.name %></p>
+            <p>
+                 <%= @user.title %>
+            </p>
+            <p><%= @user.text %></p>
+            <p>
+                <span>comments: <%= @user.comments_counter %></span>, 
+                <span>Likes: <%= @user.likes_counter %> </span>
+            </p>
+    </div>
+
+    <ul>
+          <div>
+            <% @comment.each do |item| %>
+               <p>User : <%= item.author.name %></p>
+               <p><%= item.text %></p>
+             <% end %>
+          </div>
+    </ul>
```

Comment:
- [ ] the comments for a post are not displaying on this page, as you can see in the image below, the post has 8 comments but it is not showing on the page.  kindly fix this 
on your page :point_down: 
![Screenshot from 2023-09-01 21-02-22](https://github.com/techmoves/movesBloc/assets/66097371/5b71aa8e-d27f-4fa9-bc59-047a2e402fdf)
given wireframe :point_down: 
![Screenshot from 2023-09-01 21-07-50](https://github.com/techmoves/movesBloc/assets/66097371/a5a52feb-41f5-4601-b519-928e2ca602b9)

Reply:
hi @VICTOR-CHUKWU like this
![image](https://github.com/techmoves/movesBloc/assets/110924992/f3b48078-301d-456a-9a2c-eccf62f92df7)

Reply:
- Well resolved